packages/blockchain-wallet-v4-frontend/src/modals/Nfts/NftOrder/Status/index.tsx

```diff
@@ -174,6 +200,43 @@ const NftOrderStatus: React.FC<Props> = (props: any) => {
           </ButtonWrapper>
         </>
       )}
+      {props.orderFlow.status === NftOrderStatusEnum.POST_BUY_ORDER_SUCCESS && (
+        <>
+          <Wrapper>
+            <img
+              style={{
+                borderRadius: '8px',
+                height: '64px',
+                marginRight: '12px',
+                padding: '1em',
+                width: 'auto'
+              }}
+              alt='nft-asset'
+              src={props.data.image_url}
+            />
+            <Text size='24px' weight={600}>
+              Buy Successful For
+            </Text>
+            <Text size='24px' weight={600}>
+              {props.data.name}
+            </Text>
+          </Wrapper>
+          <ButtonWrapper>
+            <Button
+              nature='primary'
+              jumbo
+              onClick={goToMyPortfolio}
```

Comment:
we don't need a new function for each status, just do onClick={props.close} and delete `goToMyPortfolio` and `closeAndViewItem` and `returnToMarketPlace `

Reply:
there also analytics that im adding to these functions, and for example goToMyPortfolio will be a route to their address page.  just closing for now